src/Components/Auth/SignUp/signup.css

```diff
@@ -0,0 +1,91 @@
+/* body {
+  height: 100vh;
+  background-image: linear-gradient(130deg, #fefefe, #ededed);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+} */
+
+.container-signup {
+    display: flex;
+    width: 800px;
+    border: 1px solid var(--black-color);
+    box-shadow: 8px 8px 0 0 var(--black-color);
+    position: relative;
+    margin: 3rem auto;
+}
+
+.left,
+.right {
+    width: 400px;
```

Comment:
changes: usually we give units more than 15px in rems unless we need exact lengths.